index.html

```diff
@@ -410,10 +411,10 @@ <h4 data-i18n="index.page.requirements.updates.header">Frequent Updates</h4>
         unsupported.os = "Windows XP";
         unsupported.i18nKey = "win-xp";
         unsupported.url = "http://blog.brackets.io/2014/04/18/windows-xp-support/";
-        unsupported.default = "Unfortunately, we don't support Windows XP any longer. <a href='" + unsupported.url + "'>Read our blog post</a> for further information.";
-    } else if (osMatch = navigator.userAgent.match(/Mac OS X 10[._](\d+)([._](\d+))?\D/i)) { // Mac OS X 10.*
-        osMatch = parseFloat(osMatch[1] + "." + (osMatch[3] || 0));
-        if (osMatch < 6.8 && osMatch !== 6.0) { // Firefox reports only the OS X major version (10.6), so we have to check for 6.0
+        unsupported.defaultText = "Unfortunately, we don't support Windows XP any longer. <a href='" + unsupported.url + "'>Read our blog post</a> for further information.";
+    } else if (osMatch = navigator.userAgent.match(/Mac OS X 10[._](\d+)(?:[._](\d+))?\D/i)) { // Mac OS X 10.*
+        osMatch = parseFloat(osMatch[1] + "." + (osMatch[2] || 0));
+        if (osMatch < 6.8 && osMatch !== 6.0) { // Firefox only reports the OS X major version (10.6), so we have to check it's not 6.0
```

Comment:
IDK, to me if feels like this area needs some whitespace, and change this long comments to sit above the condition, not inline. IMO, inline comments should only be used for two or three word comments.